### PR TITLE
fix: apply usage_unit to AWS usage type related widgets

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Please review in order by commits

### CODE REVIEW

**Review by piggggggggy — APPROVED** [review]
LGTM!

File: apps/web/src/common/composables/amcharts5/xy-chart-helper.ts [hunk 1]
```diff
@@ -230,15 +230,15 @@ export const createXYColumnSeries = (
...
     tooltip.label.adapters.add('text', (text, target) => {
         let _text = `[${gray[700]}]{valueX}[/]`;
         chart.series.each((s) => {
             const fieldName = s.get('valueYField') || s.get('valueXField') || '';
             let value = target.dataItem?.dataContext?.[fieldName];
             if (value === undefined) value = '--';
-            if (currency) value = currencyMoneyFormatter(value, currency, currencyRate);
-            _text += `\n[${s.get('stroke')?.toString()}; fontSize: 10px]●[/] [fontSize: 14px;}]${s.get('name')}:[/] [bold; fontSize: 14px]${value}[/]`;
+            const formatted = valueFormatter ? valueFormatter(value, target.dataItem?.dataContext) : value;
+            _text += `\n[${s.get('stroke')?.toString()}; fontSize: 10px]●[/] [fontSize: 14px;}]${s.get('name')}:[/] [bold; fontSize: 14px]${formatted}[/]`;
```
Comment (yuda110) [inline apps/web/src/common/composables/amcharts5/xy-chart-helper.ts:241]: praise: 👍 much better..

File: apps/web/src/services/dashboards/widgets/cost-widgets/aws-cloud-front-cost/AWSCloudFrontCostWidget.vue [hunk 2]
```diff
@@ -250,28 +261,28 @@ const drawChart = (chartData) => {
...
                 if (value === undefined) value = '--';
-                if (fieldName === 'data-transfer.out') {
+                if (typeof value === 'number') {
                     if (selectedSelectorType.value === 'cost') {
-                        if (state.currency) value = currencyMoneyFormatter(value, widgetState.currency);
+                        if (widgetState.currency) value = currencyMoneyFormatter(value, widgetState.currency);
                     } else {
-                        value = bytes.parse(`${value}${USAGE_SOURCE_UNIT}`);
-                        value = byteFormatter(value);
+                        value = `${numberFormatter(value)}${unit ? ` (${unit})` : ''}`;
```
Comment (sulmoJ) [inline apps/web/src/services/dashboards/widgets/cost-widgets/aws-cloud-front-cost/AWSCloudFrontCostWidget.vue:278]: Question: If there is no `usage_unit`, the table in costAnalysis does not show the value at all, but does the widget show the value even if there is no `usage_unit`?

**Review by sulmoJ — APPROVED** [review]
I left a question.

**Review by yuda110 — APPROVED** [review]
LGTM!!

**Pull request merged**